packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -269,31 +270,28 @@ function WorklistDataGrid(props) {
 
   // useEffect for row selection
   useEffect(() => {
+    if (!rowSelectionEffectTriggered.current) {
+      rowSelectionEffectTriggered.current = true;
+      return;
+    }
+
```

Comment:
When the consumer loads the data grid with row selection turned on for the first time, should we announce that row selection is turned on?
The requirement does not specify this scenario and we don't think this issue is isolated this JIRA or row selection. The question applies if the consumer loads with some rows selected or cells selected. Do we expect to announce messages for each one of them? @eawww, thoughts?